### PR TITLE
[SHARE-698][Task] Update model diagram

### PR DESCRIPTION
**Pull request by chrisseto**

### CODE REVIEW

File: docs/harvesters_and_transformers.rst [hunk 1]
```diff
@@ -162,10 +144,11 @@ Writing a source.yaml file
 The ``source.yaml`` file contains information about the source itself, and one or more configs that describe how to harvest and transform data from that source.
 
 .. code-block:: yaml
+
     name: com.example
     long_title: Example SHARE Source for Examples
     home_page: http://example.com/
-    user: providers.com.example
+    user: sources.com.example
```
Comment (aaxelb) [inline docs/harvesters_and_transformers.rst]: Are we going to update existing users to `sources.*`, too?

Comment (chrisseto) [inline docs/harvesters_and_transformers.rst]: I don't think it's worth updating existing users. I just wanted to remove the term provider from the docs entirely.

**Comment by aaxelb** [comment]
:+1:

File: docs/harvesters_and_transformers.rst (hunk 1 above)
Comment (aaxelb) [inline docs/harvesters_and_transformers.rst]: 👍

**Comment by leb2dg** [comment]
👏

**Pull request merged**